poppy/wfe.py

```diff
@@ -33,7 +35,7 @@ def _accept_wavefront_or_meters(f):
 
     @wraps(f)
     def wrapper(*args, **kwargs):
-        if not isinstance(args[1], Wavefront):
+        if not ( isinstance(args[1], FresnelWavefront) or isinstance(args[1], Wavefront)):
```

Comment:
I think the right fix here is to use the superclass:  `if not isinstance(args[1], BaseWavefront)`  (along with adding an import of `BaseWavefront` above.)  

@douglase, I can go ahead and make that change myself and we can discard this PR, or you can change the PR for it. Either way is fine with me so just let me know how you'd like to proceed.

Reply:
@douglase did you get a chance to see my comment last week on this? Let me know your preference and I'll work on this today or tomorrow.

Reply:
I didn't see it earlier, change made.